src/genie/libs/parser/iosxr/tests/ShowBfdSession/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,153 @@
+expected_output= {
+    "interface": {
+        "Gi0/0/0/15.103": {
+            "dest_ip_address": {
+                "172.16.100.130": {
+                    "async": "6s",
```

Comment:
let's convert to msec all the time and have only number.
```suggestion
                    "async_msec": "6000",
```

Reply:
all fields now converted to msec value and include msec in the group name.